extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1547,6 +1553,29 @@ export class AmpA4A extends AMP.BaseElement {
    */
   emitLifecycleEvent(unusedEventName, opt_extraVariables) {}
 
+  tryExecuteRealTimeConfig_() {
+    if (!!AMP.realTimeConfigManager) {
+      try {
+        return AMP.realTimeConfigManager(
+            this.element, this.win, this.getAmpDoc(),
```

Comment:
recommend this signature be !BaseElement, macros from which you can get the element, win, ampdoc